examples/acceptFromIDs.js

```diff
@@ -1,6 +1,16 @@
-var SteamUser = require('steam-user');
-var client = new SteamUser();
-var TradeOfferManager = require('steam-tradeoffer-manager');
+const SteamUser = require('steam-user'),
```

Comment:
Decide on `const` vs. `var`. You're using `const` at the top, but `var` later, which is not a fantastic way to do it. 

IMO it might also be better to follow the other examples even more closely and just use `var` for everything, and have one `var` per module instead of using commas.